examples/leds_shell/Makefile

```diff
@@ -0,0 +1,52 @@
+####
+#### Sample Makefile for building applications with the RIOT OS
+####
+#### The example file system layout is:
+#### ./application Makefile
+#### ../../RIOT
+####
+
+# Set the name of your application:
+APPLICATION = leds_shell
+
+# If no BOARD is found in the environment, use this default:
+BOARD ?= native
+
+# This has to be the absolute path to the RIOT base directory:
+RIOTBASE ?= $(CURDIR)/../../
+
+# Uncomment this to enable scheduler statistics for ps:
+#CFLAGS += -DSCHEDSTATISTICS
+
+# If you want to use native with valgrind, you should recompile native
+# with the target all-valgrind instead of all:
+# make -B clean all-valgrind
+
+# Uncomment this to enable code in RIOT that does safety checking
+# which is not needed in a production environment but helps in the
+# development process:
+#DEVELHELP = 1
+
+# Change this to 0 to show compiler invocation lines by default:
+QUIET ?= 1
+
+# Modules to include:
+USEMODULE += shell
+USEMODULE += periph_gpio
+
+# If your application is very simple and doesn't use modules that use
+# messaging, it can be disabled to save some memory:
+
+#DISABLE_MODULE += core_msg
+
+#INCLUDES += -Iapplication_include
+
+# Specify custom dependencies for your application here ...
+# APPDEPS = app_data.h config.h
+
+include $(RIOTBASE)/Makefile.include
+
+# ... and define them here (after including Makefile.include,
+# otherwise you modify the standard target):
+#proj_data.h: script.py data.tar.gz
+#	./script.py
```

Comment:
And here. If you think that documentation for this is lacking, I'd propose to add it to the documentation instead (or have a dedicated example that actually showcases the usage).